src/components/AutocompleteDropdown.tsx

```diff
@@ -193,6 +194,7 @@ function AutocompleteDropdown({
       try {
         newOptions = await onSearch(label, abortController.signal)
       } catch (error) {
+        Sentry.captureException(error)
```

Comment:
Prob want to move this inside the `if` below. This error will throw to abort the http request which will happen if the user continues to type.